robot/impl/local_robot_test.go

```diff
@@ -2409,16 +2411,18 @@ func TestCheckMaxInstanceValid(t *testing.T) {
 	cfg := &config.Config{
 		Services: []resource.Config{
 			{
-				Name:      "fake1",
-				Model:     resource.DefaultServiceModel,
-				API:       motion.API,
-				DependsOn: []string{framesystem.InternalServiceName.String()},
+				Name:                "fake1",
+				Model:               resource.DefaultServiceModel,
+				API:                 motion.API,
+				DependsOn:           []string{framesystem.InternalServiceName.String()},
+				ConvertedAttributes: &motionBuiltin.Config{},
 			},
 			{
-				Name:      "fake2",
-				Model:     resource.DefaultServiceModel,
-				API:       motion.API,
-				DependsOn: []string{framesystem.InternalServiceName.String()},
+				Name:                "fake2",
+				Model:               resource.DefaultServiceModel,
+				API:                 motion.API,
+				DependsOn:           []string{framesystem.InternalServiceName.String()},
+				ConvertedAttributes: &motionBuiltin.Config{},
```

Comment:
Previously this had a nil pointer in the ConvertedAttributes field which violates the assumptions of the motion service.